src/views/JobView.vue

```
@@ -0,0 +1,89 @@
+<script setup>
```

ESLint

[error] The 'lang' attribute of '<script>' is missing.

(vue/block-lang)